develop/app/Database/Seeds/Product.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\Database\Seeds;
+
+use CodeIgniter\Database\Seeder;
+use Faker\Factory;
+
+class Product extends Seeder
+{
+    public function run()
+    {
+        $faker = Factory::create();
+        $now   = date("Y-m-d H:i:s");
+
+        for ($i = 0; $i < 50; $i++) {
+            $data = [
+                'name'       => $faker->words(1, true),
+                'price'      => $faker->numberBetween(500, 1000),
+                'amount'     => $faker->numberBetween(10, 500),
+                'created_at' => $faker->dateTimeBetween('-2 month', '-1 days')->format('Y-m-d H:i:s'),
+                'updated_at' => $now,
+            ];
+            $this->db->table('product')->insert($data);
+
+            $orderData =[
```

Comment:
Does the order need fake data? 
@monkenWu what do you think? I think that only `production` need the fake data.

Reply:
This chahges is done.